src/assets/animations/emoji_sunglasses.json

```diff
@@ -0,0 +1 @@
55},"t":16,"s":[248.882,206.005,0],"to":[0,0,0],"ti":[-0.167,-3.75,0]},{"t":24,"s":[249.882,228.505,0]}],"ix":2,"l":2},"a":{"a":0,"k":[-0.118,-21.495,0],"ix":1,"l":2},"s":{"a":0,"k":[100,100,100],"ix":6,"l":2}},"ao":0,"shapes":[{"ty":"gr","it":[{"ind":0,"ty":"sh","ix":1,"ks":{"a":0,"k":{"i":[[0,-1.949],[0,0],[2.728,0],[0.13,-2.598],[47.547,0.909],[2.793,13.121],[2.209,0.065],[0,0],[0.455,0.195],[0.519,0],[0,0],[0.455,-2.144],[48.326,-0.909],[0.779,14.875],[2.598,0],[0,2.728],[0,0],[-1.819,0.779],[-18.252,-6.235],[-5.586,0.13],[-0.52,-0.26],[-0.585,0],[-12.081,4.157],[-27.541,-11.237]],"o":[[0,0],[0,2.728],[-2.598,0],[-0.779,14.875],[-48.326,-0.909],[-0.455,-2.144],[0,0],[-0.519,0],[-0.455,0.195],[0,0],[-2.209,0.065],[-2.793,13.121],[-47.547,0.909],[-0.13,-2.598],[-2.728,0],[0,0],[0,-1.949],[27.541,-11.237],[12.081,4.157],[0.585,0],[0.52,-0.26],[5.586,0.13],[18.252,-6.235],[1.819,0.779]],"v":[[137.911,-55.91],[137.911,-44.218],[132.975,-39.281],[128.103,-34.67],[73.476,24.569],[9.236,-33.5],[4.624,-37.268],[1.311,-37.463],[-0.118,-37.787],[-1.546,-37.463],[-4.859,-37.268],[-9.471,-33.5],[-73.711,24.569],[-128.338,-34.67],[-133.21,-39.281],[-138.146,-44.218],[-138.146,-55.91],[-135.223,-60.456],[-29.347,-61.496],[-1.807,-56.819],[-0.118,-56.494],[1.571,-56.819],[29.112,-61.496],[134.988,-60.456]],"c":true},"ix":2},"nm":"Path 1","mn":"ADBE Vector Shape - Group","hd":false},{"ty":"fl","c":{"a":0,"k":[0,0,0,1],"ix":4},"o":{"a":0,"k":100,"ix":5},"r":1,"bm":0,"nm":"Fill 1","mn":"ADBE Vector Graphic - Fill","hd":false},{"ty":"tr","p":{"a":0,"k":[0,0],"ix":2},"a":{"a":0,"k":[0,0],"ix":1},"s":{"a":0,"k":[100,100],"ix":3},"r":{"a":0,"k":0,"ix":6},"o":{"a":0,"k":100,"ix":7},"sk":{"a":0,"k":0,"ix":4},"sa":{"a":0,"k":0,"ix":5},"nm":"Transform"}],"nm":"Group 1","np":2,"cix":2,"bm":0,"ix":1,"mn":"ADBE Vector Group","hd":false}],"ip":0,"op":180,"st":0,"ct":1,"bm":0},{"ddd":0,"ind":7,"ty":4,"nm":"face","sr":1,"ks":{"o":{"a":0,"k":100,"ix":11},"r":{"a":0,"k":0,"ix":10},"p":{"a":0,"k":[249.882,237.765,0],"ix":2,"l":2},"a":{"a":0,"k":[-0.118,-12.235,0],"ix":1,"l":2},"s":{"a":0,"k":[100,100,100],"ix":6,"l":2}},"ao":0,"shapes":[{"ty":"gr","it":[{"ind":0,"ty":"sh","ix":1,"ks":{"a":0,"k":{"i":[[0,-93.271],[93.271,0],[0,93.271],[-93.271,0]],"o":[[0,93.271],[-93.271,0],[0,-93.271],[93.271,0]],"v":[[168.765,-12.235],[-0.118,156.647],[-169,-12.235],[-0.118,-181.118]],"c":true},"ix":2},"nm":"Path 1","mn":"ADBE Vector Shape - Group","hd":false},{"ty":"fl","c":{"a":0,"k":[0.992156863213,0.807843148708,0.298039227724,1],"ix":4},"o":{"a":0,"k":100,"ix":5},"r":1,"bm":0,"nm":"Fill 1","mn":"ADBE Vector Graphic - Fill","hd":false},{"ty":"tr","p":{"a":0,"k":[0,0],"ix":2},"a":{"a":0,"k":[0,0],"ix":1},"s":{"a":0,"k":[100,100],"ix":3},"r":{"a":0,"k":0,"ix":6},"o":{"a":0,"k":100,"ix":7},"sk":{"a":0,"k":0,"ix":4},"sa":{"a":0,"k":0,"ix":5},"nm":"Transform"}],"nm":"Group 1","np":2,"cix":2,"bm":0,"ix":1,"mn":"ADBE Vector Group","hd":false}],"ip":0,"op":180,"st":0,"ct":1,"bm":0}],"markers":[]}
```

Comment:
let's change path to `src/assets/lottie-files/emoji/*`. And also, rename file to `emoji-sunglasses.json` because we are using `kebab-case` for file naming